tests/test_loops.py

```diff
@@ -302,6 +302,23 @@ def func(i):
         output_names_with_port = ["output:0"]
         self.run_test_case(func, feed_dict, input_names_with_port, output_names_with_port, rtol=1e-06)
 
+    @check_tf_min_version("2")
+    @skip_tflite("shape inference fails with tflite")
```

Comment:
Could you please open an issue to describe this and put the link here for a record?

Reply:
created here: #2209